staging/src/k8s.io/apimachinery/pkg/util/net/interface.go

```diff
@@ -268,22 +331,33 @@ func (_ networkInterface) Interfaces() ([]net.Interface, error) {
 	return net.Interfaces()
 }
 
-func chooseHostInterfaceFromRoute(inFile io.Reader, nw networkInterfacer) (net.IP, error) {
-	routes, err := getRoutes(inFile)
+// getAllDefaultRoutes obtains IPv4 and IPv6 default routes on the node. If unable
```

Comment:
The IPv4 route file is a mandatory file, whereas the IPv6 route file is optional, and may not exist on the system. If we cannot read the former, it's an error. If we cannot read the latter, we ignore that error. Hopefully that is expressed clearly in the 2nd and 3rd sentences.  Don't confuse reading of the file, with the existence of routes in the files, which is addressed by the 4th sentence.